src/components/atoms/ReviewCardCarousel/ReviewCard.module.scss

```diff
@@ -0,0 +1,35 @@
+.card-container {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  justify-content: center;
+  width: 100%;
+}
+
+.card-img {
+  border: 5px solid #c33; /* Shortened the hex color */
+  border-radius: 100px; /* Removed the percentage unit for border-radius */
+}
+
+.card-heading {
+  color: #2a435d;
```

Comment:
Same changes here. Define a color variable for this component, similar to the others in foundations/sass/_colors.scss.